timemachine/fe/free_energy.py

```diff
@@ -554,12 +554,12 @@ def get_water_sampler_params(initial_state: InitialState) -> NDArray:
     nb_ixn_pot = get_bound_potential_by_type(initial_state.potentials, NonbondedInteractionGroup)
     water_params = np.array(nb_ixn_pot.params)
 
-    # If the protein is present, use the original protein parameters for the water sampler
-    if len(initial_state.protein_idxs):
-        prot_params = get_bound_potential_by_type(initial_state.potentials, Nonbonded).params[
-            initial_state.protein_idxs
-        ]
-        water_params[initial_state.protein_idxs] = prot_params
+    # If the system contains a host, use the parameters of the Nonbonded potential for the water sampler
+    # the NonbondedInteractionGroup parameters may be modified for enhanced sampling purposes
```

Comment:
Might have misunderstood, but I thought we were planning to assert that water parameters are identical in the all pairs and interaction group potentials, so this seems redundant?